compiler/one-cmds/one-import-onnx

```diff
@@ -75,13 +75,19 @@ class TidyIONames:
                     name_alt = name.replace('.', '_')
                     name_alt = name_alt.replace(':', '_')
                     self.remap_inputs.append(name_alt)
+                if name[:1].isdigit():
+                    self.input_nodes.append(name)
+                    self.remap_inputs.append('a_' + name)
         for idx in range(0, len(self.onnx_model.graph.output)):
             name = self.onnx_model.graph.output[idx].name
             if '.' in name or ':' in name:
                 self.output_nodes.append(name)
                 name_alt = name.replace('.', '_')
                 name_alt = name_alt.replace(':', '_')
                 self.remap_outputs.append(name_alt)
+            if name[:1].isdigit():
+                self.output_nodes.append(name)
+                self.remap_outputs.append('a_' + name)
```

Comment:
What if both conditions are met? For example, if output name is "3.", self.output_nodes will be ["3_", "a_3."]. I guess that it should be ["a_3_"]

Reply:
Good point. I think two condition blocks won't work as expected for `3.`...
`self.output_nodes` would have to `3.` items and two different remap outputs...
I'll merge two conditions.

Reply:
Actually, `replace` can be always applied, so if-statement seems unnecessary.

```python
              name_alt = name.replace('.','_').replace(':','_')
              if name[:1].isdigit():
                  name_alt = 'a_' + name_alt
              self.output_nodes.append(name)
              self.remap_outputs.append(name_alt)
```